test/gui/tst_activity/test.feature

```diff
@@ -0,0 +1,40 @@
+Feature: filter accounts
+
+    As a user
+    I want to filter the accounts
+    So that I can view the specific category data only
```

Comment:
```suggestion
    As a user
    I want to filter the accounts
    So that I can view the activity of each user
```

Imo, as we're filtering user accounts, we're more concerned with the activity of each individual user

Reply:
```suggestion
    As a user
    I want to filter resources
    So that I can view the specific category resources only
```